### PR TITLE
Fix strict cert issue on Python 2.7.9 plus added support of bucket names

### PR DESCRIPTION
**Pull request by aguereca**
Fix strict cert issue on Python 2.7.9 plus added support of bucket names with dots.
Reference:
https://github.com/boto/boto/issues/2836

### CODE REVIEW

**Comment by metaodi** [comment]
@aguereca I guess you're refering to https://github.com/boto/boto/issues/2836, correct?

File: sync.py [hunk 1]
```diff
@@ -21,15 +25,31 @@ class SyncS3():
...
+                    aws_access_key_id=self.AWS_ACCESS_KEY,
+                    aws_secret_access_key=self.AWS_SECRET_KEY,
+                    calling_format=OrdinaryCallingFormat())
+            else:
+                conn = connect_to_region(
+                    self.REGION,
+                    aws_access_key_id=self.AWS_ACCESS_KEY,
+                    aws_secret_access_key=self.AWS_SECRET_KEY)
+
+        return conn.get_bucket(self.BUCKET_NAME, validate=False)
```
Comment (metaodi) [inline sync.py]: Why are you using `validate=False`? According [to the docs](http://boto.readthedocs.org/en/latest/ref/s3.html):

> This is only safe to do if you are **sure** the bucket exists.

Comment (aguereca) [inline sync.py]: @metaodi Yes, this is about boto/boto#2836 Good catch on the validate=False , was a test that I made and not intended to part of the PR.

**Comment by metaodi** [comment]
Thanks @aguereca for this PR!

**Pull request merged**